service/actions.go

```diff
@@ -214,7 +215,7 @@ var notifyUpdateServiceInstance = action.Action{
 	MinParams: 4,
 }
 
-type bindPipelineArgs struct {
+type bindAppPipelineArgs struct {
```

Comment:
I think this could also be a generic struct, instead of bindAppPipelineArgs we could do something like:

```
in: binder.go we could add all the other methods necessary for bind.App interface, this is just so you get the idea
type BindableObj interface {
	GetName() string
	GetPool() string
}

type bindAppPipelineArgs struct {
	bindableObject             bind.BindableObj
	writer          io.Writer
	serviceInstance *ServiceInstance
	appParams          BindAppParameters
	event           *event.Event
	requestID       string
	shouldRestart   bool
	forceRemove     bool
}
```